keycloak_api_client/api_client.py

```diff
@@ -74,19 +74,34 @@ def _get_client_mappers_url(self, id_of_client: UUID) -> str:
     def _get_authorization_header(self) -> str:
         return f'Bearer {self._get_api_admin_oidc_token()}'
 
-    def _get_api_admin_oidc_token(self) -> str:
+    def _get_api_admin_oidc_token(
```

Comment:
@BartoszJarosz there is a lot of usage class state. That could cause some issues and it's less readable. I would suggest to cut some logic from that function so it always takes `client_id` and `client_secret` as an obligatory parameters. That will cause moving logic of picking required values to functions instead of that more magic like approach.